src/encrypt-decrypt.cpp

```diff
@@ -0,0 +1,60 @@
+#include "encrypt-decrypt.h"
+
+using namespace std;
+
+char curr;
+
+// Random key to use for encryption/decryption
+int key = 214748;
+
+void encrypt(fs::path &input, fs::path &output) {
```

Comment:
The functions should in theory encrypt or decrypt the file you pass it. I don't see the need for two paths.

Reply:
That was my implementation initially but I added the two paths for increased flexibility and to make the process "non-destructive". I can change it back to in-place encryption/decryption if necessary though.

Reply:
I would recommend it as it means we don't need to come up with secondary names for files.

Reply:
I've changed the functions to work in-place: 3e0d2eb